netlify.toml

```diff
@@ -1,6 +1,6 @@
 [build]
-  base      = "site/"
-  publish   = "site/dist"
+  base      = "/"
```

Comment:
We should handle this separately, but this is the Netlify default so we should be able to get rid of this. Separate PR though 😊